source/web_integration/index.rst

```diff
@@ -6,8 +6,10 @@ Web Integration dynamically displays a unique phone number for each source drivi
 See the following technical documentation for details on how to setup Web Integration:
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 4
```

Comment:
I don't think you want to increase the depth. This is not the number of files listed below, but the number of levels that will be in the table of contents.